Tests/When_parsing_links.cs

```diff
@@ -16,6 +16,17 @@ public void Should_return_url_in_canonical_form_for_GitHub(string provided)
         }
 
         [Theory]
+        [InlineData("git@gitlab.com:ctaggart/sourcelink-test.git")]
+        [InlineData("https://gitlab.com/ctaggart/sourcelink-test.git")]
+        [InlineData("https://gitlab.com/ctaggart/sourcelink-test")]
+        [InlineData("https://gitlab.com/ctaggart/sourcelink-test/")]
+        public void Should_return_url_in_canonical_form_for_GitLab(string provided)
```

Comment:
Might be worth adding local GitLab urls as well to the tests?

Reply:
like also test against a `gitlab.example.com` domain?